src/cunumeric/unary/unary_red_omp.cc

```diff
@@ -98,6 +98,34 @@ struct UnaryRedImplBody<VariantKind::OMP, OP_CODE, CODE, DIM> {
   }
 };
 
+template <UnaryRedCode OP_CODE, Type::Code CODE, int DIM>
+struct UnaryRedImplBodyWhere<VariantKind::OMP, OP_CODE, CODE, DIM> {
+  using OP    = UnaryRedOp<OP_CODE, CODE>;
+  using LG_OP = typename OP::OP;
+  using RHS   = legate_type_of<CODE>;
+
+  void operator()(AccessorRD<LG_OP, true, DIM> lhs,
+                  AccessorRO<RHS, DIM> rhs,
+                  AccessorRO<bool, DIM> where,
+                  const Rect<DIM>& rect,
+                  const Pitches<DIM - 1>& pitches,
+                  int collapsed_dim,
+                  size_t volume) const
+  {
+    Splitter<DIM> splitter;
+    auto split = splitter.split(rect, collapsed_dim);
+
+#pragma omp parallel for schedule(static)
+    for (size_t o_idx = 0; o_idx < split.outer; ++o_idx)
```

Comment:
Minor nit, but consider adding the `{}` around the `for`'s body. The code is correct as written, just makes it slightly easier to read.